conjur/login/login_logic.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginLogic module
+
+This module is the business logic for writing user-specific configuration
+information to the user's machine
+"""
+
+# Builtins
+import logging
+
+# Internals
+import conjur.constants
+from conjur.endpoints import ConjurEndpoint
+from conjur.http import invoke_endpoint, HttpVerb
+
+class LoginLogic:
+    """
+    LoginLogic
+
+    This class holds the business logic for populating the
+    netrc configuration details needed to login to Conjur
+    """
+    credentials = None
+
+    def __init__(self, credentials):
+        self.credentials = credentials
+
+    @classmethod
```

Comment:
Why static class method?

Reply:
doesn't use `self` so doesn't need to be a regular method

Reply:
ssl_verify leaves in the controller but has no usage instead of passing it to logic.
maybe it should be part of logic instead?
controller should be stateless as possible